app/src/main/java/com/kickstarter/viewmodels/ProjectSearchResultHolderViewModel.java

```diff
@@ -18,113 +18,102 @@
 
 public interface ProjectSearchResultHolderViewModel {
 
-  final class Data {
```

Comment:
using a `Pair` helped us clean up a lot of boilerplate code. in general, using a `Data` class for view models is nice, but typically for when there are 3+ pieces of data is when it really gives us a win